### ISSUE TITLE
Parsed data is no longer available

### ISSUE THREAD

**Issue by romanbsd** [voltrue2/in-app-purchase#58]
It was available on 0.7.3, but not any longer.
For example, for code 21006:
```javascript
{ status: 21006,
     message: 'This receipt is valid but the subscription has expired. When this status code is returned to your server, the receipt data is also decoded and returned as part of the response.' }
```
The receipt and latest_expired_receipt_info are no longer available, since in apple.js:117 and apple.js:158 validatedData is set to {status,message} losing the returned data entirely.

**Comment by voltrue2** [comment]
Hello

Thank you for reaching out and reporting this issue.
I will have a look at it as soon as I get a chance.

Cheers

**Comment by voltrue2** [comment]
Hello, 

I have added all response from Apple to the returned validation data: f3db2ea245a7627c4b82d7e11eca18c4cde7f813
Published version is 1.1.7.

Thanks again for reporting it.

Cheers

**Comment by romanbsd** [comment]
👍
